src/wallet/scriptpubkeyman.h

```diff
@@ -481,9 +493,13 @@ class LegacyScriptPubKeyMan : public ScriptPubKeyMan, public FillableSigningProv
     void LearnAllRelatedScripts(const CPubKey& key);
 
     /**
-     * Marks all keys in the keypool up to and including reserve_key as used.
+     * Marks all keys in the keypool up to and including the provided key as used.
+     *
+     * @param keypool_id determines the last key to makr as used
```

Comment:
In fec5ded232ca37b8a965ce09a3940ccb6ce43249 "Return used destinations from ScriptPubKeyMan::MarkUnusedAddresses"

typo: makr - > mark

```suggestion
     * @param keypool_id determines the last key to mark as used
```

Reply:
Done